app/routes.py

```
@@ -749,35 +917,44 @@ async def meetingbaas_webhook(request: Request):
-            logger.info(f"Call ended event detected for bot {bot_id}, generating summary...")
+            logger.info(
+                f"Call ended event detected for bot {bot_id}, generating summary..."
+            )
 
             # Try to find the transcript file
             # First try with the MeetingBaaS bot_id, then look for any matching internal client_id
             transcript_dir = os.path.join(get_state_dir(), "transcripts")
             transcript_file = None
 
-            if bot_id and os.path.exists(os.path.join(transcript_dir, f"{bot_id}.json")):
+            if bot_id and os.path.exists(
```

Ruff (0.15.20)

[warning] Async functions should not use os.path methods, use trio.Path or anyio.path

(ASYNC240)